docs/CustomApp.md

```diff
@@ -52,9 +52,17 @@ import { UserList, UserEdit, UserCreate } from './User';
 // your app labels
 import messages from './i18n';
 
+const i18nProvider = (type, params) => {
+    if (type === 'CHANGE_LOCALE') {
+        return messages[params.locale];
+    }
+    throw new Error('Undefined action type', type); 
```

Comment:
this won't work, the initial message is `GET_DEFAULT_MESSAGES` and it will be rejected. Besides, isn't it `GET_LOCALE_MESSAGES`?